__testHelpers__/test.helper.js

```diff
@@ -0,0 +1,27 @@
+// You may obtain a copy of the License at
```

Comment:
This header should be the BC Gov header.

Reply:
Header added